type-annotator-scanner/src/test/java/edu/ucr/cs/riple/scanner/tools/TrackerNodeDisplay.java

```diff
@@ -59,7 +63,7 @@ public int hashCode() {
 
   @Override
   public String toString() {
-    return "callerClass='"
+    return "\ncallerClass='"
```

Comment:
This is used mainly in tests and that is because we print the list of `unexpected results` via a call to `ArrayList.toString()` and that writes its content with format `[item1.toString(), item2.toString(),...]` which in long strings, its hard to detect which character is different when comparing with the displayed expected results. With this implementation, it will be displayed as:
```java
[
item1.toString(), 
item2.toString(),
...]
```

The other alternative is to append `\n` to end of the `toString()` but that will print it as below:
```java
[item1.toString,
item2.toString(),
]
```

which I think the current implementation shows a better view of the content and is easier to detect the sequence causing the error. Please let me know if you prefer to change this.

Reply:
Or we could use something other than `ArrayList.toString()` when printing unexpected results, perhaps even using `String.join('\n', ...)`. I just feel it's cleaner in terms of separation of concerns if `toString()` isn't overly specific about how or where it expects to be called. 🙂

Reply:
Sure, sounds good and it's much cleaner now, thanks b2ad4afa5547865ada4b5e9ee42e6bacf496dc0d.